### PR TITLE
Milestone 2.7: Remove documentation related to protractor.

### PR DESCRIPTION
**Pull request by ShivamJhaa**
This PR removed the following documentation:
* Hybrid State
* Guide to migrate end to end tests
* Debug End to End tests in protractor
* End to End tests in protractor

### CODE REVIEW

File: Debug-custom-ESLint-check-tests.md [hunk 1]
```diff
@@ -36,15 +36,15 @@ DEBUGGING found lint error
...
       });
 DEBUGGING found lint error
 DEBUGGING found lint error
       ✓ it('should test a feature', function() {
-        element(by.css('.protractor-test')).click();
+        $('.webdriverio-test')).click();
       });
       ✓ it('should test a feature', function() {
-        element(by.css('.protractor-test')).sendKeys("keys");
+        $('.webdriverio-test')).setValue("keys");
```
Comment (U8NWXD) [inline Debug-custom-ESLint-check-tests.md]: Isn't this now `.e2e-test`?

**Review by U8NWXD — CHANGES_REQUESTED** [review]
just one nit

**Review by SAEb-ai — APPROVED** [review]
Except U8NWXD's comment above, LGTM. Thanks and congratulations for completing the full migration finally.

File: Debug-custom-ESLint-check-tests.md (hunk 1 above)
Comment (ShivamJhaa) [reply Debug-custom-ESLint-check-tests.md]: Yes, corrected.

**Comment by ShivamJhaa** [comment]
@U8NWXD PTAL, thanks!

**Comment by ShivamJhaa** [comment]
> Except U8NWXD's comment above, LGTM. Thanks and congratulations for completing the full migration finally.

Thanks @SAEb-ai .

**Review by U8NWXD — APPROVED** [review]

**Pull request merged**